config.yaml

```diff
@@ -18,4 +18,18 @@ server6:
         # implement (i)PXE boot
         - pxeboot: tftp://[2001:db8::1]/ipxe/x86_64/ipxe http://[2001:db8::1]/ipxe/boot6
         # create Endpoint objects in kubernetes
-        - metal: inventory.yaml
\ No newline at end of file
+        - metal: metal_config.yaml
+
+---
+# metal_config.yaml
```

Comment:
Do we want to have this config here or in an example config folder?

Reply:
Fair point, split and moved to an `example` folder